### PR TITLE
add to readme how to work with bash, and reset tools

### PR DESCRIPTION
**Pull request by brachi-wernick**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,7 +157,7 @@ To make sure everything works well, run `docker ps`. You should see 4 containers
      4.3 If you wish to stop producing data, you can stop the containers using `docker-compose stop datagen` or `docker-compose stop producer`
 
 5. Check if the consumer shows the stream output successfully:
-  run `docker logs kafka-consumer -f` or `docker-compose logs -f consumer`.
+  run `docker-compose logs kafka-consumer -f` or `docker-compose logs -f consumer`.
```
Comment (hcloli) [inline README.md]: Not true. either docker logs kafka-consumer or docker-compose logs consumer. If you are using docker-compose, than it is by service name. If docker - it is by container name.

File: README.md [hunk 2]
```diff
@@ -166,6 +167,35 @@ MESSAGE=> key:user_3, value:LoginCount{userName='user_3', count=1, windowStart=1
 MESSAGE=> key:user_4, value:LoginCount{userName='user_4', count=1, windowStart=1545346145000, windowEnd=1545346146000}
 ```  
 
+6. To open kafka bash run `docker exec -i -t kafka /bin/bash`
+
+Here, you can run Kafka tools from CLI, for example creating a topic
+
+`$KAFKA_HOME/bin/kafka-topics.sh --create --topic my-topic  --zookeeper zookeeper:2181 --partitions 1 --replication-factor 1`
+
+Tow important things to look into in the above command:
```
Comment (hcloli) [inline README.md]: Two instead of "tow".

File: README.md [hunk 3]
```diff
@@ -166,6 +167,35 @@ MESSAGE=> key:user_3, value:LoginCount{userName='user_3', count=1, windowStart=1
...
+- I use `$KAFKA_HOME` to access Kafka bin tools
```
Comment (hcloli) [inline README.md]: Not needed. You can use the commands without prefixing with KAFKA_HOME. They are in the path.

File: README.md [hunk 4]
```diff
@@ -233,3 +263,9 @@ as well and select the left and right values wisely - simple and small as possib
 In most cases, the better way to output data in Kafka Stream is to write to topic. The output information is 
 usually needed in sub-systems such as elasticsearch, SQL DB etc. Consider using kafka connect or a simple 
 consumer application to copy the data into the sub-system.
+
+
+
+docker exec -i -t kafka /bin/bash
```
Comment (hcloli) [inline README.md]: What is this? Remove these lines. They have nothing to do here.

**Review by hcloli — CHANGES_REQUESTED** [review]

**Comment by brachi-wernick** [comment]
@hcloli ready for code review, I'll work on the elastic connector/another simple connector in another PR

**Review by hcloli — APPROVED** [review]

**Pull request merged**